src/main/tv/codely/scala_intro_examples/lesson_06_anon_fn_currying/exercise_solutions/daniel_albarral_logger.sc

```diff
@@ -0,0 +1,22 @@
+import java.util.{Calendar, Date}
+
+
+def clock(calendar: Calendar) = calendar.getTime
```

Comment:
Instead of using the `java.util.Calendar` you also can use the [already installed](https://github.com/CodelyTV/scala-intro-examples/blob/master/build.sbt#L10) [nscala-time library](https://github.com/nscala-time/nscala-time):

```scala
import com.github.nscala_time.time.Imports._

DateTime.now()
```